module/input/Camera/data/config/igus2/Cameras/Left.yaml

```diff
@@ -1,20 +1,20 @@
 driver: Aravis
-deviceID: 00F7D6EC
+deviceID: 0118847F
 
 buffer_count: 4             # The number of software buffers to use.
 
 format:
   width: 1280
   height: 1024
-  pixel: BayerBG8
+  pixel: BayerRG8
 
 lens:
-  radiansPerPixel: 0.00245
+  radiansPerPixel: 0.0024242              # 1/focal_length_in_pixels = camera_pixel_size / camera_focal_length = 4.8um / 1.98mm = 2.4242e-3
```

Comment:
Do it properly!

Reply:
https://medium.com/@kennethjiang/calibrate-fisheye-lens-using-opencv-333b05afa0b0

Reply:
No

Reply:
Seems you have committed other things then!

Reply:
THAT'S ONLY SO THEY'RE ALL THE SAME :P

Reply:
Calibration should be a separate PR

Reply:
Happy now!